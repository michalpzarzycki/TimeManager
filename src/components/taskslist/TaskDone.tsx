import React from 'react';
import styles from './TaskDone.module.css';

export default function TaskDone({task, setDoneTask, setUncompletedTask}: any) {
    return (<>
        <div className={styles.id}>
            {task.done ? <button onClick={() => setDoneTask(task)}>DONE</button> : <button onClick={() => setUncompletedTask(task)}>Not done</button>}
        </div>
        <div className={task.done ? styles.done : styles.notDone}></div>
        </>
    )
}