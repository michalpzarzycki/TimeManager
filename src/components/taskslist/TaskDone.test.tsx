import React from 'react'
import {shallow} from 'enzyme'
import TaskDone from './TaskDone'

test('Expext to render TaskDone component', () => {
    expect(shallow(<TaskDone task={{}} setDoneTask={{}} setUncompletedTask={{}}/>)).toMatchSnapshot()
})
