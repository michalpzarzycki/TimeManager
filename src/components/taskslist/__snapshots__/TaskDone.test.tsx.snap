// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expext to render TaskDone component 1`] = `ShallowWrapper {}`;
